admin/banner_statistics.php

```diff
@@ -178,7 +178,7 @@
           });
 
           function showTooltip(x, y, contents) {
-              $('<div id="tooltip">' + contents + '</div>').css({
+              $('<div id="tooltip">' + contents + '<\/div>').css({
```

Comment:
Are you sure? Why?

Reply:
Surprise, surprise:

![image](https://user-images.githubusercontent.com/4391026/208125487-071db695-20c0-4603-8b75-80167b444bed.png)

Reply:
thanks for clarifying.